lib/apiservers/portlayer/restapi/handlers/interaction_handlers.go

```diff
@@ -146,7 +147,7 @@ func (i *InteractionHandlersImpl) UnbindHandler(params interaction.InteractionUn
 // ContainerResizeHandler calls resize
 func (i *InteractionHandlersImpl) ContainerResizeHandler(params interaction.ContainerResizeParams) middleware.Responder {
 	// See whether there is an active session to the container
-	session, err := i.attachServer.Get(context.Background(), params.ID, 0)
+	session, err := i.server.Interaction(context.Background(), params.ID, interactionTimeout)
```

Comment:
The timeout here was set to `0` previously because we were finding resize getting called either at somewhat arbitrary times and it was adding a 30s delay on the CLI end (IIRC it was the stop path - this may have changed given the 1.13 use of events, but needs confirming).